blocks/footer/footer.css

```diff
@@ -1,14 +1,451 @@
+}
+
+/* Footer Bottom : Legal */
+
+.ft-legal {
+  display: grid;
+  grid-template-columns: 1fr 1.5fr;
+  justify-content: space-between;
+  border-top: 1px var(--primary-blue-80) solid;
+  padding: 1.5rem 0 2rem;
   margin: auto;
+  font-size: 14px;
+  line-height: 16.8px;
+  font-weight: 400;
+  
+  .ft-legal-list {
+    & p {
+      margin: 0;
+      padding: 0;
+    }
+    
+    &:last-child {
+      & ul {
+        display: flex;
+        flex-wrap: nowrap;
+        justify-content: space-between;
+        
+        & a {
+          text-decoration: none;
+          color: var(--color-white);
+        }
+      }
+    }
+  }
+}
+
+/* Tablet */
+
+@media only screen and (width <= 1024px) {
+  footer {
+    padding: 10px 50px;
+  }
+
+  /* Footer Top : Parent DIV contains : (Logo, Brands, Social links) */
+  .ft-social {
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: 1fr;
+    grid-gap: 1.5rem;
+    
+    & .ft-social-list  { /* Footer Top : ITEMS (Logo, Brands, Social links) */
+     
+      &:last-child { /* Footer Top : Social Links */
+        grid-column: 1 / 3;
+        grid-row: 2 / 2;
+     
+        & ul {
+          justify-content: space-evenly;
+          padding: 1rem 0 0;
+          border-top: 1px solid var(--primary-blue-80);
+        }
+      }
+    }
+  }
+  
+  /* Footer Main */
+  .ft-main {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+    grid-gap: 0 1.5rem;
+    padding:  2rem 0;
+    
+    & .ft-main-item {
+      padding-bottom: 2rem;
+      
+      &:last-child {
+        grid-column: 1 / 4;
+        grid-row: 2 / 2;
+        border: 0;
+        border-top: 1px solid var(--primary-blue-80);
+        padding: 3rem 0 0;
+        width: 100%;
+
+        & ul {
+          display: flex;
+          justify-content: space-between;
+          flex-direction: row;
+          align-items: center;
+        }
+      }
+
+      & a {
+        padding: 0;
+      }
+    }
+  }
+
+  html[dir="rtl"] .ft-main-item:last-child {
+    border-left: 0;
+    border-right: 0;
+    padding-top: 1.5rem;
+  }
+
+  html[dir="rtl"] .ft-main-item:first-child {
+    padding-right: 0;
+  }
+
+  /* Footer Bottom : Legal */
+  .ft-legal {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    justify-content: space-between;
+    grid-row-gap: 0;
+    padding:  2rem 0;
+    
+    & ul li {
+      margin: 0 1rem;
+    }
+
+    & .ft-legal-list {
+      &:first-child {
+        display: flex;
+        flex-wrap: nowrap;
+        justify-content: center;
+        padding: 0 0 1rem;
+      } 
+
+      &:last-child {
+        & ul {
+          display: flex;
+          flex-wrap: wrap;
+          justify-content: center;
+          margin: 0;
+          padding: 0;
+
+          & li {
+            padding: .5rem 0;
+            margin: 0 1rem;
+          }
+        }
+      } 
+    }
+  }
 }
 
-footer .footer p {
-  margin: 0;
+/* Mobile */
+@media only screen and (width <= 768px) {
```

Comment:
Implement mobile first approach, write global rules for mobile and override for desktop.
Also mobile view is <768px not <=768

Reply:
@mkbansal1 Just an FYI, in responsive design, header and footer are usually exempt from being mobile-first approach as we have lots of elements in desktop for header and footer and it makes sense to have them first and then hide those in mobile. There's no right or wrong here. Both should work.

Reply:
Agree, global rules should be outside any media query, also px size should be corrected